### ISSUE TITLE
Show numbers for fullscreen spaces

### ISSUE THREAD

**Issue by Hritik14** [xiamaz/YabaiIndicator#16]
Currently it only shows `F` for fullscreen spaces. Can it start counting all spaces including fullscreen ?

<img width="305" alt="image" src="https://github.com/xiamaz/YabaiIndicator/assets/7457065/680f784a-e68f-4361-9041-d3e0e81ccbd4">

**Comment by xiantang** [comment]
Yes. Because I switch spaces using cmd + number, not including fullscreen in the calculation would cause confusion
